scripts/python/tao_build.py

```diff
@@ -377,10 +377,9 @@ def _write(line, cmd="build"):
         _write("--host_jvm_args=-Djdk.http.auth.tunneling.disabledSchemes=", cmd = "startup")
         logger.info("configuring tao_bridge with bazel ......")
 
-    proxy_setting = "HTTPS_PROXY= " if args.platform_alibaba else ""
     with cwd(tao_bazel_root), gcc_env(args.bridge_gcc):
         # make sure version.h is generated
-        execute(f"{proxy_setting} bazel build --config=release //:version_header_genrule")
+        execute(f"bazel build --config=release //:version_header_genrule")
```

Comment:
Why should the proxy be removed?

Reply:
The original code disabled proxy in internal in deed.